src/middlewares/setInstance.js

```diff
@@ -0,0 +1,13 @@
+const { logger } = require("../utils/logger");
+
+module.exports = function () {
+    return setInstance=(req, res, next) => {
+        if (req.headers['x-openhim-clientid']) {
+          const clientId = req.headers['x-openhim-clientid'];
+            res.locals.instanceValue = clientId;
+        } else {
+          res.status(500).send({ error: "Required Headers were not passed"})
```

Comment:
We should be returning a [400](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400)

Reply:
Updating.